data/releases.yml

```diff
@@ -323,6 +323,30 @@
             - name: Install via crates.io
               link: https://crates.io/crates/skywalking
 
+    - name: SkyWalking PHP
+      icon: sw-agent
+      description: The PHP Agent for Apache SkyWalking, which provides the native tracing abilities for PHP project.
+      source:
+        - version: v0.1.0
+          date: Oct. 08th, 2022
+          downloadLink:
+            - name: tar
+              link: https://www.apache.org/dyn/closer.cgi/skywalking/php/0.1.0/skywalking_agent-0.1.0.tgz
+            - name: asc
+              link: https://downloads.apache.org/skywalking/php/0.1.0/skywalking_agent-0.1.0.tgz.asc
+            - name: sha512
+              link: https://downloads.apache.org/skywalking/php/0.1.0/skywalking_agent-0.1.0.tgz.sha512
+      distribution:
+        - version: v0.1.0
+          date: Oct. 08th, 2022
+          downloadLink:
+            - name: tar
+              link: https://www.apache.org/dyn/closer.cgi/skywalking/php/0.1.0/skywalking_agent-0.1.0.tgz
```

Comment:
Distribution is binary download. We need to link to PECL?

Reply:
No, PECL only works on the `pecl install skywalking_agent` command

Reply:
Also can be `pecl install https://www.apache.org/dyn/closer.cgi/skywalking/php/0.1.0/skywalking_agent-0.1.0.tgz`.

Reply:
So, there is no binary distribution?

Reply:
<img width="481" alt="image" src="https://user-images.githubusercontent.com/5441976/194686815-78fc713c-731c-4b5e-bafb-14aaf39d405b.png">

You could refer this style, and link to your official doc